packages/dashboard/src/dashboard-actions/reverseAction.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { MockDashboardFactory, MockWidgetFactory, MOCK_EMPTY_DASHBOARD } from '../testing/mocks';
+import { applyReverseAction, reverseAction } from './reverseAction';
+//broken into sections based on reverse actions and applying reversals.
+//broken into sub sections based on move or resize
+describe('reverseActions - MOVE', () => {
```

Comment:
entire file is in the context of `reverse`, describe block better stated as "move action"